tutor/src/components/multi-select.jsx

```diff
@@ -72,28 +106,51 @@ class MultiSelect extends React.Component {
         onSelect={this.onSelect}
         className="multi-selection-option"
       >
-        <Icon type={selection.selected ? 'check-square' : 'square'} />
+        <Icon variant={selection.selected ? 'checkedSquare' : 'checkSquare'} size="lg" />
         <span className="title">{selection.title}</span>
         {onlyToggle}
       </Dropdown.Item>
     );
   };
 
+  renderHelperControls = () => {
+    if (!this.props.showHelperControls) { return; }
+
+    return (
+      <div className="multi-select-helpers">
+        <a href="#" className="select-all" onClick={this.onSelectAll}>All</a>
+        <span className="divider">|</span>
+        <a href="#" className="select-none" onClick={this.onSelectNone}>None</a>
+      </div>
+    )
+  }
+
   render() {
     return (
-      <Dropdown variant="default" className={classnames('multi-select', this.props.className)}>
+      <Dropdown
+        variant="default"
+        className={classnames('multi-select', this.props.className)}
+        onToggle={this.onToggle}
+        show={this.isOpen}
+      >
         <Dropdown.Toggle
           id="multi-select"
           variant="default"
           aria-label={this.props.title}
-          onToggle={this.onToggle}
-          open={this.isOpen}
           tabIndex={this.props.tabIndex}
         >
           {this.props.title}
         </Dropdown.Toggle>
         <Dropdown.Menu>
-          {Array.from(this.props.selections).map((selection) => this.renderMenuSelection(selection))}
+          <MultiSelectWrapper
+            useColumns={this.props.useColumns}
+            showHelperControls={this.props.showHelperControls}
+          >
+            {this.renderHelperControls()}
+            <MultiSelectItems useColumns={this.props.useColumns}>
+              {Array.from(this.props.selections).map((selection) => this.renderMenuSelection(selection))}
+            </MultiSelectItems>
+          </MultiSelectWrapper>
         </Dropdown.Menu>
       </Dropdown>
     );
```

Comment:
This is fine, but something that would be a bit cleaner would be to add a `helperControls` prop instead of the `showHelperControls` bool.  Doing so is more flexible since you could then pass alternative controls if needed.

You'd then extract the `onSelectAll` and `onSelectNone` methods and `renderHelperControls` into a MultiSelectHelperControls component  and pass it to multiSelect like:

```js
<Multiselect 
  helperControls={< MultiSelectHelperControls onSelectAll={...} />}
/>
```

I don't think it's worth changing to this right now, but wanted to be aware of the technique.